python/src/trezorlib/transport/webusb.py

```diff
@@ -114,15 +115,21 @@ def get_path(self) -> str:
         return f"{self.PATH_PREFIX}:{dev_to_str(self.device)}"
 
     @classmethod
-    def enumerate(cls, usb_reset: bool = False) -> Iterable["WebUsbTransport"]:
+    def enumerate(
+        cls, models: Optional[Iterable["TrezorModel"]] = None, usb_reset: bool = False
```

Comment:
Isn't it better to use the following?
```suggestion
        cls, models: Iterable["TrezorModel"] = TREZORS, usb_reset: bool = False
```
and drop the `if models is None: models = ...` below?

Reply:
Same as above